admin/acceptance.py

```diff
@@ -106,6 +106,13 @@ def start_nodes(self):
             cwd=self.vagrant_path.path,
             env=extend_environ(FLOCKER_BOX_VERSION=box_version))
 
+        from flocker.provision._install import run, task_pull_docker_images
+        for node in self.NODE_ADDRESSES:
+            run(
+                username='root',
+                address=node,
+                commands=task_pull_docker_images()
```

Comment:
But each of the other examples do including pulling the image before hand.

http://doc-dev.clusterhq.com/gettingstarted/examples/environment.html#download-the-docker-image

I think (given an infinite, or longer timeout), all the tests would pass without this (but I haven't tested that). I think the only reason only one test fails due to this, is that only one test happens to trigger twisted timeout code. (I think it only happens when you return a deferred, and only for the time that deferred takes; the tests are mostly synchronous).

Perhaps it makes sense to move this logic elsewhere in the acceptance tests, but I'm not entirely sure where the best place is for it. (And if that is the case, all the other providers should get the same change). I can file an issue for doing this, but I think this fix is sufficient for the moment.